workflows/sarek.nf

```diff
@@ -128,6 +128,10 @@ if (params.tools && (params.tools.split(',').contains('ascat') || params.tools.s
     }
 }
 
+if ((params.download_cache) && (params.snpeff_cache || params.vep_cache)) {
+    error("Please specify either `--download_cache` or `--snpeff_cache`, `--vep_cache`.\nhttps://nf-co.re/sarek/dev/usage#how-to-customise-snpeff-and-vep-annotation")
```

Comment:
did you update these docs already? Quick question how does this work for a cluster wide setup? I am thinking it would make sense to instruct users to predownload this, same as for igenomes and then supply the cache location (maybe sarek specific config specific config, to keep download costs low in terms of time and storage space for users)

Reply:
yeah, for a regular HPC, that depends if you build yourself or if you copy annotation_cache.
If you build locally, then no issue, just set up `use_annotation_cache_keys` to false.
If just basic mirror of annotation_cache then `use_annotation_cache_keys` should be set to true.
But for now, no other options.